Lookuptable/testCases/test_lookuptable_creation.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+
+import pytest
+
+from Lookuptable.testPages.data.export_data_page import ExportDataPage
+from common_utilities.Excel.excel_manage import ExcelManager
+from Lookuptable.testPages.data.lookup_table_page import LookUpTablePage
+from common_utilities.generate_random_string import fetch_random_string
+from Lookuptable.userInputs.user_inputs import UserData
+
+""""Contains test cases related to the Data module"""
+
+values = dict()
+
+@pytest.mark.data
+@pytest.mark.managetables
+def test_19_create_lookup_table(driver):
+    data = LookUpTablePage(driver)
+    export = ExportDataPage(driver)
+    export.data_tab()
+    data.create_lookup_table()
+
+@pytest.mark.data
+@pytest.mark.viewtables
+def test_29_view_lookup_table(driver):
+    data = LookUpTablePage(driver)
+    export = ExportDataPage(driver)
+    export.data_tab()
+    data.create_lookup_table()
+    data.view_lookup_table()
+    data.delete_lookup_table()
+
+
+@pytest.mark.data
+@pytest.mark.managetables
+def test_37_select_deselect(driver):
+    export = ExportDataPage(driver)
+    export.data_tab()
+    data = LookUpTablePage(driver)
+    data.selects_deselects()
+
+@pytest.mark.data
+@pytest.mark.managetables
+def test_38_edit_table(driver):
+    data = LookUpTablePage(driver)
+    export = ExportDataPage(driver)
+    export.data_tab()
+    data.edit_table()
+
+@pytest.mark.data
+@pytest.mark.managetables
+def test_39_create_dummy_id(driver):
+        data = LookUpTablePage(driver)
+        export = ExportDataPage(driver)
+        export.data_tab()
+        data.create_dummyid()
+
+@pytest.mark.data
+@pytest.mark.managetables
+def test_40_edit_dummy_data(driver):
+        data = LookUpTablePage(driver)
+        export = ExportDataPage(driver)
+        export.data_tab()
+        data.edit_dummy_data()
+
+@pytest.mark.data
+@pytest.mark.managetables
+@pytest.mark.lookupexcel
+def test_21_Error_upload3(driver):
+    export = ExportDataPage(driver)
+    export.data_tab()
+    data = LookUpTablePage(driver)
+    values['table_id'] = data.create_download_lookuptable()
+    Download_path = data.error_upload1()
+    print("path is " + Download_path)
+    data.write_data_excel(values['table_id'], Download_path)
+    data = LookUpTablePage(driver)
```

Comment:
You have already created an object of LookUpTablePage in line 72. You can remove this one. One instance is enough per test.